### ISSUE TITLE
write_videofile output mp4 does not have the same color as the original mp4

### ISSUE THREAD

**Issue by doctor-yes** [Zulko/moviepy#1794]
This is my code

#files_mp4 is the paths to multiple source files
```
    for file in files_mp4: 
        thisFile=os.path.join(path,file)
        output.append(VideoFileClip(thisFile))

    final_clip = concatenate_videoclips(output)
    final_clip.write_videofile("test.mp4", audio_fps=48000,  ffmpeg_params=['-c:v', 'libx265'])
```

The output mp4 color is slightly "yellower" than original, as you can see from the yellow text and the green grass. This happens to every video and I can't seem to find any previous note about this. What is wrong and how can I fix this? Thanks.

![image](https://user-images.githubusercontent.com/16641322/174470594-6d5a66f8-2af5-43f7-92c9-1346c70ddb32.png)

**Comment by 6Leoo6** [comment]
I have a similar problem. I have an image with grey background and I'm trying to insert it into the main video. Everything looks good in preview but after writing to file the grey background is turned into black (with default libx264). Then I tried using the png codec and it returned the true colour of the image but with 130x the size (It's a 20m long video so the output was 48 Gb) of the video with mp4 or libx264. It was unacceptably huge so I tried converting it with VLC and it worked! Nothing has changed with the quality of the video and the size only increased around 10-20 Mbs. I wanted to automate a process with moviepy and it took forever to render the video and the manually convert it with VLC. So is there a better approach only using python? This is the basic VLC codec:
![kép_2022-06-19_135151488](https://user-images.githubusercontent.com/77102589/174479476-899e13ad-31b7-400d-a576-250fb114909e.png)
.
